### PR TITLE
Release/3.6.6

### PR DESCRIPTION
**Pull request by supercid**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 All notable changes to this project will be documented in this file. This project adheres to Semantic Versioning.
 
+### 3.6.6
```
Comment (chenliangde) [inline CHANGELOG.md]: It deserves a 3.7.0.

Comment (chenliangde) [reply CHANGELOG.md]: Let's add the change log about the programatic tagging here

File: app/code/community/Nosto/Tagging/Block/Category.php [hunk 2]
```diff
@@ -51,15 +51,13 @@ protected function _toHtml()
...
-            || !$this->getCategory()
+        if (!$helper->existsAndIsConnected()
+            || $this->getCategory() === null
+            || !Mage::helper('nosto_tagging/module')->isModuleEnabled()
         ) {
             return '';
         }
-
-        return parent::_toHtml();
+        return (new Nosto_Object_Category($this->getCategory()))->toHtml();
```
Comment (chenliangde) [inline app/code/community/Nosto/Tagging/Block/Category.php]: It seems that there is no way to override the category in the mini nosto plugin. At least in prestashop world many merchants are doing so.

Comment (supercid) [reply app/code/community/Nosto/Tagging/Block/Category.php]: There is, we need to override the category block in the mini plugin
```
<?xml version="1.0"?>
<config>
    <modules>
        <My_Nosto>
            <version>0.1.0</version>
        </My_Nosto>
    </modules>
    <global>
        <blocks>
            <nosto_tagging>
                <rewrite>
                    <category>My_Nosto_Block_Category</category>
                </rewrite>
            </nosto_tagging>
        </blocks>
    </global>
</config>
```

```php
<?php
// app/code/local/My/Nosto/Block/Category.php
/**
 * Overridden category block.
 */
class My_Nosto_Block_Category extends Nosto_Tagging_Block_Category
{
    public function _toHtml()
    {
        return '<html>here</html>';
    }
}

```

**Pull request closed without merging**